src/pytestqt/exceptions.py

```diff
@@ -98,14 +98,6 @@ class TimeoutError(Exception):
     .. versionadded:: 2.1
 
     Exception thrown by :class:`pytestqt.qtbot.QtBot` methods.
-
-    .. note::
-        In versions prior to ``2.1``, this exception was called ``SignalTimeoutError``.
-        An alias is kept for backward compatibility.
     """
 
     pass
-
-
-# backward compatibility alias
-SignalTimeoutError = TimeoutError
```

Comment:
I was wondering this days if it wouldn't make sense to ditch our custom `TimeoutError` and raise the builtin `TimeoutError` instead... what do you think?

Reply:
Python's `TimeoutError` is a `OSError` subclass, and documented as:

> Raised when a system function timed out at the system level. Corresponds to errno `ETIMEDOUT`.

So that doesn't really seem like a good match for us. Additionally, this would cause [more churn than I'm comfortable with](https://github.com/search?q=pytestqt.exceptions+-extension%3Ajson+-extension%3Aebuild&type=Code) for something we already changed one time before, so I don't think it's worth it.

Reply:
Fair enough. 👍